build.gradle

```diff
@@ -211,7 +211,19 @@ subprojects {
                 exclude "**/*Proto.java"
                 // warnings are errors
                 options.addBooleanOption('Xwerror', true)
-                options.overview = "src/main/javadoc/overview.html"
+                options.with {
+                    overview "src/main/javadoc/overview.html"
+
+                    // Make our Javadocs link to external resources
```

Comment:
It would probably be nice to have how to generate the package list files, or a link to whatever documentation you referred to.

Reply:
Okay, rather than document it, I added a script that does it and a description instructing people to run the script.